csv/src/test/java/org/apache/metamodel/csv/SingleLineCsvDataSetTest.java

```diff
@@ -64,4 +70,84 @@ public void testMalformedLineParsing() throws Exception {
         assertFalse(ds.next());
         ds.close();
     }
-}
\ No newline at end of file
+
+    public void testCustomTyping() throws Exception {
+        ColumnType columnType1 = ColumnType.DATE;
```

Comment:
Same here as above. In my opinion just using the constant is more clear than adding it to a variable which doesn't add a meaning to it.